spec/lucky_web/input_helpers_spec.cr

```diff
@@ -154,3 +168,9 @@ end
 private def view
   TestPage.new
 end
+
+def have_unchecked_value(value)
```

Comment:
Oops. I forgot the `private` part in my example. Could you mark this is `private` (like on the method above) so it doesn't leak into other specs?